### ISSUE TITLE
docfx: add a way to easily go from YAML to HTML

### ISSUE THREAD

**Issue by tbpg** [googleapis/doc-templates#115]
We need a single command for people to run to take a directory of YAML and convert it to HTML.

**Comment by tbpg** [comment]
This might actually fit inside of doc-pipeline. That's where we generate the `docfx.json` file and run `docfx`. So, maybe we should transfer this over there? "Easy way to run doc-pipeline without needing to upload anything to Cloud Storage."

**Comment by dandhlee** [comment]
I agree that this would work better in doc-pipeline; albeit it can be done, making use of the process that already exists in doc-pipeline would make this a lot more easier.

Will reopen this in doc-pipeline.
